### DISCUSSION TITLE
Ask for Refund

### DISCUSSION THREAD

**Discussion by deanchou1306** [aclap-dev/video-downloadhelper#927] [License & Purchase requests]
### What's your issue?

Dear Admin. The downloader doesn't work. Please refund me.

My Account: dinhmmo@hotmail.com

The 10 first digits (only 10) of your license number or email address

dinhmmo@hotmail.com

Thank so much

### The 10 first digits (only 10) of your license number or email address

dinhmmo@hotmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.

**Comment by deanchou1306** [comment]
ok, i got it, thank you
